contract_compassion/models/contracts.py

```diff
@@ -218,32 +218,6 @@ def _compute_contract_duration(self):
     ##########################################################################
     #                              ORM METHODS                               #
     ##########################################################################
-    @api.model
```

Comment:
Actually you don't have to remove this code, it makes the tests fail. It won't do anything since you already set the commitment_number in sponsorship_compassion, and when you call super it will come here and do nothing because of the line 223 that tests if commitment_number is already set.